src/RobotInterface/YarpImplementation/src/YarpSensorBridge.cpp

```diff
@@ -80,24 +80,12 @@ bool YarpSensorBridge::initialize(std::weak_ptr<IParametersHandler> handler)
         std::cout << logPrefix << " Skipping the configuration of CartesianWrenches. YarpSensorBridge will not stream relevant measures." << std::endl;
     }
 
-    bool useCameras{false};
-    ret = m_pimpl->subConfigLoader("stream_cameras", "Cameras",
-                                    &YarpSensorBridge::Impl::configureCameras,
-                                    handler,
-                                    m_pimpl->metaData,
-                                    useCameras);
-    if (!ret)
-    {
-        std::cout << logPrefix << " Skipping the configuration of Cameras. YarpSensorBridge will not stream relevant measures." << std::endl;
-    }
-
-
     m_pimpl->bridgeInitialized = true;
     return true;
 }
 
 bool YarpSensorBridge::setDriversList(const yarp::dev::PolyDriverList& deviceDriversList)
-{
+{    
```

Comment:
White spaces

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/182/commits/d4dcb8a28a7f4b4a59fee3f7b96e2f25f6a4d268.